### PR TITLE
Allow to use selector when listing clusterroles

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://github.com/openshift/origin/issues/12450

Basically pass the ListOptions when filtering the policies.

### CODE REVIEW

**Comment by mfojtik** [comment]
@liggitt @smarterclayton PTAL

File: pkg/authorization/registry/role/policybased/virtual_storage.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func (m *VirtualStorage) NewList() runtime.Object {
 }
 
 func (m *VirtualStorage) List(ctx kapi.Context, options *kapi.ListOptions) (runtime.Object, error) {
-	policyList, err := m.PolicyStorage.ListPolicies(ctx, options)
```
Comment (liggitt) [inline pkg/authorization/registry/role/policybased/virtual_storage.go]: is the same change needed for RoleBindings?

**Comment by liggitt** [comment]
would like to see tests exercising this

File: pkg/authorization/registry/role/policybased/virtual_storage.go (hunk 1 above)
Comment (mfojtik) [inline pkg/authorization/registry/role/policybased/virtual_storage.go]: yeah, I will check and add some tests

**Comment by mfojtik** [comment]
@liggitt updated.

[test]

**Comment by liggitt** [comment]
LGTM, [merge]

**Comment by mfojtik** [comment]
@jmazzitelli I believe `delete` and `get` are using the same selector code underneath, but I will verify it.

**Pull request merged**